### PR TITLE
created SkillFamilyPage and SkillFamilyTable

### PR DESCRIPTION
**Pull request by JamesHuf**
Created SkillFamilyPage for #1681

### CODE REVIEW

**Review by petertgiles — CHANGES_REQUESTED** [review]
Wow, turned this one around fast!
Could you please add a storybook story for the new table?  That might require you to make a fake data generator as well.

File: admin/resources/js/components/PoolDashboard.tsx [hunk 1]
```diff
@@ -289,6 +297,14 @@ export const PoolDashboard: React.FC = () => {
         description: "Label displayed on the Departments menu item.",
       })}
     />,
+    <MenuLink
+      key="skill-family"
```
Comment (yonikid15) [inline admin/resources/js/components/PoolDashboard.tsx:301]:
```suggestion
      key="skill-families"
```

**Review by yonikid15 — CHANGES_REQUESTED** [review]
Great work James 👍. Just found one typo that I made a suggestion for.

File: admin/resources/js/components/PoolDashboard.tsx (hunk 1 above)
Comment (JamesHuf) [reply admin/resources/js/components/PoolDashboard.tsx]: Thanks! It's fixed now.

**Comment by petertgiles** [comment]
Heads up, I merged main for your branch.

File: frontend/admin/resources/js/api/skillFamilyOperations.graphql [hunk 2]
```diff
@@ -0,0 +1,30 @@
+query AllSkillFamilies {
+  skillFamilies {
```
Comment (petertgiles) [inline frontend/admin/resources/js/api/skillFamilyOperations.graphql:2]: The SkillFamily model has a `key` field that needs to be returned as well.

File: frontend/admin/resources/js/components/skillFamily/SkillFamilyTable.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        }),
+        accessor: (sf) => sf.description?.[locale],
+      },
+      {
+        Header: intl.formatMessage({
+          defaultMessage: "Category",
+          description:
+            "Title displayed for the Skill Family table Category column.",
+        }),
+        accessor: "category",
```
Comment (petertgiles) [inline frontend/admin/resources/js/components/skillFamily/SkillFamilyTable.tsx:54]: I think this should be localized as well.  It's an enum so check the localizedstrings files for examples.

**Review by petertgiles — CHANGES_REQUESTED** [review]
Nice, this is looking great!  Just a few small things to check.  Also, I see that description is not populating properly but I see that's an issue with the seeder.  It seems to work fine in the storybook story.  I'll submit an issue for that.

**Review by petertgiles — APPROVED** [review]
I merged in Tristan's French toggle PR which has caused a conflict here.  Everything looks good to me.  The page is crashing due to that seeder issue but I could verify it works on my system after fixing it.  Nice work! 👍

**Comment by petertgiles** [comment]
Fix for the seeder issue: https://github.com/GCTC-NTGC/gc-digital-talent/pull/1862

**Pull request merged**